agent/engine/docker_task_engine.go

```diff
@@ -758,3 +786,25 @@ func (engine *DockerTaskEngine) Capabilities() []string {
 func (engine *DockerTaskEngine) Version() (string, error) {
 	return engine.client.Version()
 }
+
+// enableConcurrentPull checks the docker version and return true if docker version > 1.9.0
+func (engine *DockerTaskEngine) enableConcurrentPull() bool {
+	version, err := engine.Version()
+	if err != nil {
+		seelog.Warnf("Failed to get docker version, err %v", err)
+		return false
+	}
+
+	match, err := utils.Version(version).Matches(">=1.9.0")
```

Comment:
From this [commit](https://github.com/docker/docker/issues/9718#issuecomment-146966627),and the [PR](https://github.com/docker/docker/pull/16893) it was fixed in docker 1.9.0. I have tested with 1.9.1 which was used in the previous AMI. Will test against with all other docker versions as suggested.